symengine/cwrapper.cpp

```diff
@@ -1148,6 +1148,25 @@ CWRAPPER_OUTPUT_TYPE dense_matrix_eye(CDenseMatrix *s, unsigned long int N,
     CWRAPPER_END
 }
 
+CWRAPPER_OUTPUT_TYPE dense_matrix_diff(CDenseMatrix *result,
+                                       const CDenseMatrix *A, basic const x)
+{
+    if (not is_a_Symbol(x))
+        return SYMENGINE_RUNTIME_ERROR;
```

Comment:
A test would have been to have an integer (or anything other than a symbol) as input and check that the function returns a `SYMENGINE_RUNTIME_ERROR`.